bot/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+eessi_arch=x86_64 # will/should be set by bot
```

Comment:
Either we use `$(uname -m)` here now now, or we pick up on `${CPU_TARGET}` set by the bot in the Slurm job, see https://github.com/EESSI/eessi-bot-software-layer/pull/141 (we need to pull it apart though, since it specifies something like `linux/x86_64/intel/haswell` now)

Reply:
This build script should only actually only built when `$CPU_TARGET` is set to `*/generic`, and not do anything for other more specific CPU targets